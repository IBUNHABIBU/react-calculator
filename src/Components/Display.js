import React from 'react';

const Display = () => (
  <>
    <h1>Displaying the calculator</h1>
  </>
);

export default Display;