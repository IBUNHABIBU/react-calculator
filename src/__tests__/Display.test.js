/* globals describe, expect, it, beforeEach */
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Display from '../Components/Display';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Display result="48" />);
});

describe('Display the result', () => {
  it('should display the result of the calculation', () => {
    expect(wrapper.find('.display-container').text()).toContain('48');
  });
});

describe('snapshots', () => {
  it('renders App component correctly', () => {
    const tree = renderer.create(<Display />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
