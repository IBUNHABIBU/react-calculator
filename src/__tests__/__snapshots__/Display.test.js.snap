// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders App component correctly 1`] = `
<div
  className="display-container"
>
  0
</div>
`;
